engine/src/test/resources/Seloni.json

```diff
@@ -6,21 +6,43 @@
       "/rest/unprocess/count/DND"
     ],
     "className": "className",
+    "filepath": "I:\\code\\seloni-demo-test\\className.class",
```

Comment:
paths OS specific... will this test work in unix?

Reply:
yes. we need it just to compare the strings. no intention to execute